docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express')
```

Comment:
Can you add a license header?

Reply:
still needs a header 😄

Reply:
sorry saw a comment about header. but didn't see here